<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectrumPage</class>
 <widget class="QWidget" name="SpectrumPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SpectrumPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="75">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="8,2">
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>холст</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGraphicsView" name="canvas">
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="title">
        <string>параметры построения</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>длина отрезков</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="lengthInput">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>кол-во отрезков</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="amountInput">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>360</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>шаг (градусы)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="stepInput">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>359</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>алгоритм 1</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="algorithm1Input">
            <item>
             <property name="text">
              <string>ЦДА</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Брезенхэм действительный</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Брезенхэм целочисленный</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Брезенхэм с устранением</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Библиотечный</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>алгоритм 2</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="algorithm2Input">
            <item>
             <property name="text">
              <string>ЦДА</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Брезенхэм действительный</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Брезенхэм целочисленный</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Брезенхэм с устранением</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Библиотечный</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>цвет</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QGraphicsView" name="colorInput">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="draw1Button">
          <property name="text">
           <string>построить 1-ую серию</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="draw2Button">
          <property name="text">
           <string>построить 2-ую серию</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>очистить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
